Modules/CLI/ResampleDTIVolume/itkSeparateComponentsOfADiffusionTensorImage.h

```diff
@@ -61,14 +61,11 @@ class SeparateComponentsOfADiffusionTensorImage
 protected:
   SeparateComponentsOfADiffusionTensorImage();
 
-  void ThreadedGenerateData( const OutputImageRegionType & outputRegionForThread, ThreadIdType threadId ) ITK_OVERRIDE;
+  void DynamicThreadedGenerateData( const OutputImageRegionType & outputRegionForThread) override;
 
-  void GenerateOutputInformation() ITK_OVERRIDE;
-
-  void GenerateInputRequestedRegion() ITK_OVERRIDE;
-
-private:
```

Comment:
Is removing `private` desired ?

Reply:
Yes, it was intended, there were no members or functions after the private keyword. It should have no effect.